hazelcast/src/main/java/com/hazelcast/map/impl/operation/steps/UtilSteps.java

```diff
@@ -25,6 +25,19 @@
 
 public enum UtilSteps implements IMapOpStep {
 
+    FINAL_STEP() {
```

Comment:
So, for all tiered store operations, we will return this FINAL_STEP checking if the compaction is needed?

Reply:
we will use injected step, final is a dispatcher step.